src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -209,7 +209,7 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
         track={track}
         analyticsUseCase={analyticsUseCase}
         isNew={isNew}
-        isShareable={isLegacyLicense && !expired}
+        isShareable={!expired}
```

Comment:
think we should remove the code that adds 'coming soon' as well

Reply:
Yeah good point